crates/store/src/db/migrations/2025062000000_setup/up.sql

```diff
@@ -18,17 +18,17 @@ CREATE TABLE accounts (
     block_num                               INTEGER NOT NULL,
     account_commitment                      BLOB NOT NULL,
     code_commitment                         BLOB,
-    storage                                 BLOB,
-    vault                                   BLOB,
     nonce                                   INTEGER,
+    storage_header                          BLOB, -- Serialized AccountStorage from miden-objects
```

Comment:
Maybe I missed a comment about this somewhere, but why are we serializing full `AccountStorage` into a field named `storage_header`?

If the intention is to serialize full `AccountStorage` (as it was before), then we should probably keep this field named `storage`. If we do want to rename it to `storage_header`, we should have it store `AccountStorageHeader`.

The former is probably acceptable as it means we want to make the updates incrementally - i.e., first migrate the vault structure and then migrate the storage structure. But again, I think the name of the field should reflect its content.

Reply:
The comment is wrong, the actual change of serializing the header only is implemented.